assets/css/navbar.css

```diff
@@ -0,0 +1,197 @@
+/**
+ * Project: portfolio
+ * File: navbar.css
+ * Created: 05/02/23
+ * Author: Abdullah Al Mamun <mamun1214@gmail.com>
+ */
+
+/* .navbar {
+    position       : relative;
+    display        : flex;
+    flex-wrap      : wrap;
+    align-items    : center;
+    justify-content: space-between;
+    height         : 50px;
+}
+
+.navbar>.container {
+    display        : flex;
+    flex-wrap      : inherit;
+    align-items    : center;
+    justify-content: space-between;
+}
+
+.nav-menu-bar {
+    display: none;
+}
+
+.navbar-collapse {
+    flex-basis     : auto;
+    flex-grow      : 1;
+    align-items    : center;
+    min-width      : 768px;
+    justify-content: center;
+    display        : flex;
+}
+
+.navbar-nav {
+    display       : flex;
+    flex-direction: row;
+    padding-left  : 0;
+    margin-bottom : 0;
+    margin-top    : 0;
+    list-style    : none;
+}
+
+.nav-item {
+    margin: 0;
+}
+
+.nav-link {
+    height         : 2.5rem;
+    position       : relative;
+    color          : #172b4d;
+    display        : block;
+    padding        : 0 0.5rem;
```

Comment:
- [OPTIONAL] Kindly make sure to remove all the commented out lines of code to make your code clean :recycle: